src/components/Avatar/Avatar.component.tsx

```diff
@@ -39,6 +40,24 @@ export const CAMERA = {
     }
   }
 };
+export interface SpawnState {
+  onMountEffect?: {
+    src: string | null;
+    animationSrc?: string | null;
+    loop?: number | null;
+  } | null;
+  onMountAnimation?: {
+    src: string | null;
+    loop?: number | null;
+  } | null;
+}
```

Comment:
I think it would be better to move this under `src/types`

Reply:
thx!